scripts/play.ts

```diff
@@ -0,0 +1,402 @@
+
+    // Ensure dependencies are isolated from the monorepo.
+    if (!(await pathExists(new URL("./yarn.lock", PLAYGROUNDS_ROOT)))) {
+      await fs.writeFile(new URL("./yarn.lock", PLAYGROUNDS_ROOT), "");
+    }
+
+    await createPlayground(playgroundName, playgroundDir, {
+      framework: args.framework ?? DEFAULT_FRAMEWORK,
+      dryRun: args["dry-run"],
+    });
+  }
+
+  if (args.start) {
+    await exec("yarn", ["dev"], {
+      cwd: playgroundDir,
+      stdio: "inherit",
+      dryRun: args["dry-run"],
+    });
+  }
+
+  process.off("uncaughtException", handleUncaughtException);
+}
+
+/**
+ * Creates a new playground.
+ *
+ * @param name - The name of the playground. It must be unique.
+ * @param dir - The directory of the playground. It must be unique.
+ * @param options - Options that determine the function's behavior.
+ */
+async function createPlayground(
+  name: string,
+  dir: URL,
+  options: DryRunOption & {
+    /**
+     * The framework used to bootstrap the playground.
+     */
+    framework: Args["framework"];
+  },
+) {
+  switch (options.framework ?? "next") {
+    case "next": {
+      await cloneGitRepo(
+        "https://github.com/prismicio-community/nextjs-starter-prismic-minimal-ts.git",
+        dir,
+        { dryRun: options.dryRun },
+      );
+
+      await updatePackageJSON(
+        dir,
+        { scripts: { "next:dev": "next dev --port=8001" } },
+        { dryRun: options.dryRun },
+      );
+
+      await updateSliceMachineConfig(
+        dir,
+        { localSliceSimulatorURL: "http://localhost:8001/slice-simulator" },
+        { dryRun: options.dryRun },
+      );
+
+      break;
+    }
+
+    case "sveltekit": {
+      await cloneGitRepo(
+        "https://github.com/prismicio-community/sveltekit-starter-prismic-minimal.git",
+        dir,
+        { dryRun: options.dryRun },
+      );
+
+      break;
+    }
+
+    default: {
+      console.error(chalk.red`Framework not supported: ${options.framework}`);
+      process.exit(1);
+    }
+  }
+
+  if (!options.dryRun) {
+    await fs.rm(new URL("./.git", dir), { recursive: true });
+    await fs.rm(new URL("package-lock.json", dir));
+  }
+
+  await exec("yarn", ["add", "--dev", "cross-env"], {
+    cwd: dir,
+    stdio: "inherit",
+    dryRun: options.dryRun,
+  });
+
+  await updatePackageJSON(
+    dir,
+    {
+      name,
+      scripts: {
+        slicemachine:
+          "cross-env SM_ENV=staging ../../packages/start-slicemachine/bin/start-slicemachine.js",
+      },
+    },
+    { dryRun: options.dryRun },
+  );
+
+  await updateSliceMachineConfig(
+    dir,
+    { apiEndpoint: `https://${name}.cdn.wroom.io/api/v2` },
+    { dryRun: options.dryRun },
+  );
+
+  await exec(
+    "yarn",
+    [
+      "dlx",
+      "--quiet",
+      "@slicemachine/init@latest",
+      `--repository="${name}"`,
+      "--no-start-slicemachine",
+    ],
+    {
+      cwd: dir,
+      env: {
+        // Use wroom.io
+        SM_ENV: "staging",
```

Comment:
Should we be able to run the play script with a prod repo? I feel it can still be usefull to test, WDYT?

Reply:
Yep, you can now. I figured we could add that functionality in later, but it wasn't hard to add now. 🙂

```sh
```